hop/models.py

```diff
@@ -103,12 +139,79 @@ def from_email(cls, email_input):
 
         """
         if hasattr(email_input, "read"):
-            msg = email.message_from_file(email_input)
+            message = email.message_from_file(email_input)
         else:
-            msg = email.message_from_string(email_input)
+            message = email.message_from_string(email_input)
 
         # format gcn circular into header/body
         return cls(
-            header={title.lower(): content for title, content in msg.items()},
-            body=msg.get_payload(),
+            header={title.lower(): content for title,
+                    content in message.items()},
```

Comment:
Is there any way to get this to fit into one line instead? It's harder to see that it's a dict comprehension when it's split like this.

Reply:
Yep, good catch. That was an auto-format change which definitely makes it less readable. Fixed.